examples/csmsc/voc3/local/synthesize_e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+config_path=$1
+train_output_path=$2
+ckpt_name=$3
+
+FLAGS_allocator_strategy=naive_best_fit \
+FLAGS_fraction_of_gpu_memory_to_use=0.01 \
+python3 ${BIN_DIR}/../../synthesize_e2e.py \
+    --am=fastspeech2_csmsc \
+    --am_config=${config_path} \
+    --am_ckpt=${train_output_path}/checkpoints/${ckpt_name} \
+    --am_stat=dump/train/speech_stats.npy \
+    --voc=mb_melgan_csmsc \
+    --voc_config=mb_melgan_csmsc_ckpt_0.1.1/default.yaml \
+    --voc_ckpt=mb_melgan_csmsc_ckpt_0.1.1/snapshot_iter_1000000.pdz\
```

Comment:
the file in train_output_path is `mb_melgan_csmsc_ckpt` ?

Reply:
in file:
https://github.com/PaddlePaddle/PaddleSpeech/blob/develop/examples/csmsc/README.md

voc3 - MultiBand MelGAN